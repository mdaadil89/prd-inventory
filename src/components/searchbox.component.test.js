import React from 'react'
import {mount, shallow} from 'enzyme'
import SearchBox from './searchbox.component'
import toJson from 'enzyme-to-json';


describe('Searchbox Snapshot', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = mount(<SearchBox/>);
    });

    test('renders correctly', () => {
        expect(toJson(wrapper)).toMatchSnapshot(); 
    }); 
     
});