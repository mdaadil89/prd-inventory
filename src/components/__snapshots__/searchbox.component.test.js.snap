// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbox Snapshot renders correctly 1`] = `
<SearchBox>
  <Col
    md={3}
  >
    <div
      className="col-md-3"
    >
      <input
        name="filter"
        placeholder="search products"
        type="search"
      />
    </div>
  </Col>
</SearchBox>
`;
